modules/prometheus-exporter_oracle/conf/06-fra.yaml

```diff
@@ -0,0 +1,24 @@
+module: oracle
+name: "fast recovery area usage"
+id: fra_usage
+
+transformation: true
+aggregation: ".mean(by=['job'])"
+
+signals:
+  signal:
+    metric: oracledb_FRA_Usage_value
+    rollup: latest
+
+
+rules:
```

Comment:
warning + critical is not logic.

use critical/major couple or minor/warning one but not warning/critical